logprep/framework/pipeline_manager.py

```diff
@@ -101,18 +102,11 @@ def restart_failed_pipeline(self):
             self.set_count(self._configuration.get("process_count"))
             self._logger.warning(f"Restarted {len(failed_pipelines)} failed pipeline(s)")
 
-    def handle_logs_into_logger(self, logger: Logger, timeout: float):
-        """Handle logs."""
-        try:
-            logger.handle(self._log_handler.get(timeout))
-            while True:
-                logger.handle(self._log_handler.get(0.0))
-        except Empty:
-            pass
-
     def stop(self):
         """Stop processing any pipelines by reducing the pipeline count to zero."""
         self._decrease_to_count(0)
+        while len(self._pipelines) > 0:
```

Comment:
Will this ever be  >0? `_decrease_to_count(0)`  has basically the same condition in its while loop.
Also, if this wouldn't be always true, then it would rapidly spam the log message.